docker_test.go

```diff
@@ -237,7 +234,7 @@ func TestMainFlow(t *testing.T) {
 		messages, errors := consume(t, c, topics)
 
 		signals := make(chan os.Signal, 1)
-		signal.Notify(signals, os.Interrupt, os.Kill) // Get signal for finish
+		signal.Notify(signals, os.Interrupt, syscall.SIGTERM) // Get signal for finish
```

Comment:
[optional]

```suggestion
		signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM) // Get signal for finish
```

Reply:
I like `os.Interrupt`, it is os agnostic. Unfortunately there is no similar thing for `syscall.SIGTERM`